roles/openshift_node/tasks/install.yml

```diff
@@ -1,28 +1,15 @@
 ---
-- when: not openshift_is_containerized | bool
-  block:
-  - name: Install Node package
-    package:
-      name: "{{ openshift_service_type }}-node{{ (openshift_pkg_version | default('')) | lib_utils_oo_image_tag_to_rpm_version(include_dash=True) }}"
-      state: present
-    register: result
-    until: result is succeeded
-
-  - name: Install sdn-ovs package
-    package:
-      name: "{{ openshift_service_type }}-sdn-ovs{{ (openshift_pkg_version | default('')) | lib_utils_oo_image_tag_to_rpm_version(include_dash=True) }}"
-      state: present
-    when:
-    - openshift_node_use_openshift_sdn | bool
-    register: result
-    until: result is succeeded
-
-  - name: Install conntrack-tools package
-    package:
-      name: "conntrack-tools"
-      state: present
-    register: result
-    until: result is succeeded
+- name: Install Node package, sdn-ovs, conntrack packages
+  package:
+    name:
```

Comment:
`name: "{{ item }}"` or is this somehow implicit?

Reply:
Yeah, this patch needs some tweaking.  I'm not sure what I was doing here.